src/main/java/babysitter/BabySitter.java

```diff
@@ -33,50 +30,34 @@ boolean invalidUserEntry(int startTime, int endTime, int bedTime) {
 				if (invalids[i] == sc.nextInt()) {
 					invalidTracker = true;
 				}
-				invalidTracker = false;
 			}
 		}
 		return invalidTracker;
 
 	}
 
 	int convertAm(int am) {
-		if (am > 0 && am <= 4) {
-			am += 24;
-		}
-		return am;
+		return (am <= 4) ? am + 24 : am;
```

Comment:
Aside from refactoring this to be more succinct, there was also a functional problem: It should have been `am >= 0` instead of `am > 0` (otherwise you wouldn't adjust the midnight hour).

The statement `am >= 0 && am <=4` can be written more plainly as `am <= 4` (as long as you're not expecting negative numbers), so that led me to the ternary-style one-line return statement.